app/classifier/classification-summary.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import tasks from './tasks/index.coffee';
 import { TextSplit } from 'seven-ten';
+import tasks from './tasks';
```

Comment:
Good catch, there were a few other import errors that were causing the tests to fail.  I did not catch all of them in the rebase.  I will fix this up so at least the tests pass.